atlas/AtlasMl/atlasml/ml/MLPipelines/PipelineWorkflows.py

```
@@ -0,0 +1,278 @@
+        4. Competency-to-cluster association
+        5. Text-to-competency linking
+
+        Note:
+            - Deletes all existing cluster centers before creating new ones
+            - Updates both COMPETENCY and TEXT collections with new associations
+        """
+        texts = self.weaviate_client.get_all_embeddings(CollectionNames.TEXT.value)
+
+        # Generate embeddings for each text entry and collect UUIDs
+        embeddings_list = np.vstack([text["vector"]["default"] for text in texts])
+        embeddings_uuids = np.array([text["properties"]["text_id"] for text in texts])
```

Ruff (0.11.9)

Local variable `embeddings_uuids` is assigned to but never used

Remove assignment to unused variable `embeddings_uuids`

(F841)